### PR TITLE
Chore: Remove agg store flag

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Remove unnecessary feature flag `ENABLE_SNS_AGGREGATOR_STORE`.

# Changes

* Remove usage of feature flag in sns.store.ts
* Remove it from dfx.json
* Remove it from featureFlagsStore
* Remove usage in sns services.

# Tests

* Remove tests that checked the feature flag. I have kepts the describe when flag is true. Changing that made the diff harder to read.
* Fix env tests.
* Remove it from jest setup.

# Todos

- [ ] Add entry to changelog (if necessary).

### CODE REVIEW

File: frontend/src/tests/lib/services/sns.services.spec.ts [hunk 1]
```diff
@@ -114,10 +113,6 @@ describe("sns-services", () => {
 
   describe("loadSnsDerivedState", () => {
     describe("with ENABLE_SNS_AGGREGATOR_STORE true", () => {
```
Comment (lmuntaner) [inline frontend/src/tests/lib/services/sns.services.spec.ts:115]: The tests will be moved in another PR.

File: frontend/src/tests/lib/services/sns.services.spec.ts [hunk 2]
```diff
@@ -478,9 +328,6 @@ describe("sns-services", () => {
 
   describe("loadSnsLifecycle", () => {
     describe("with ENABLE_SNS_AGGREGATOR_STORE true", () => {
```
Comment (lmuntaner) [inline frontend/src/tests/lib/services/sns.services.spec.ts:330]: The tests will be moved in another PR.

File: frontend/src/tests/lib/stores/sns.store.spec.ts [hunk 3]
```diff
@@ -223,31 +216,7 @@ describe("sns.store", () => {
...
-          lifecycles: [SnsSwapLifecycle.Open],
-          certified: true,
-        });
-
-        snsQueryStore.setData(data);
-        expect(get(isLoadingSnsProjectsStore)).toBe(false);
-      });
-    });
-
     describe("with ENABLE_SNS_AGGREGATOR_STORE true", () => {
```
Comment (lmuntaner) [inline frontend/src/tests/lib/stores/sns.store.spec.ts:219]: The tests will be moved in another PR.

File: frontend/src/tests/lib/stores/sns.store.spec.ts [hunk 4]
```diff
@@ -525,155 +494,9 @@ describe("sns.store", () => {
...
-
-      it("does NOT use snsAggregator as source of data", () => {
-        snsAggregatorStore.setData([aggregatorSnsMockDto]);
-        snsQueryStore.reset();
-
-        expect(get(snsSummariesStore)).toHaveLength(0);
-      });
-    });
-
     describe("flag ENABLE_SNS_AGGREGATOR_STORE is enabled", () => {
```
Comment (lmuntaner) [inline frontend/src/tests/lib/stores/sns.store.spec.ts:497]: The tests will be moved in another PR.

**Comment by lmuntaner** [comment]
@dskloetd please review

Deployed to testnet: https://tcvdh-niaaa-aaaaa-aaaoa-cai.nnsdapp.dfinity.network/

**Review by dskloetd — APPROVED** [review]
Thanks for not messing up the diff by not yet ungrouping the tests!
